### ISSUE TITLE
Vignettes Not Working

### ISSUE THREAD

**Issue by nliftman** [nliftman/shapez#9]
@rudeboybert The vignettes are on our local versions, but there is no way to push them to the remote. We have tried it on both of our computers as well as on the servers on school computers. :(

**Comment by rudeboybert** [comment]
Hey Naomi and Les, I'm at a loss as to why these files are not being committed.

However, based on nothing other than comparing it to what ggplot2 does: https://github.com/tidyverse/ggplot2/tree/main/vignettes

the absence of the html files might not be a problem. I guess the thinking is you should only commit/version control source files, like .Rmd and .R files, and not the output files, like .html files. The idea being you should always keep the outputs fresh based on what's in the source files.

**Comment by rudeboybert** [comment]
honestly, the "just do what hadley wickham and jenny bryan" do approach to these things might not be the best, but it has yet to fail me so far

**Comment by nliftman** [comment]
@rudeboybert Yeah, I see that they do not have the .html files, but I am still at a loss for why the vignette is not visible on our website. I have spent quite a few hours trying to find the answers on the internet to no avail. Will we get points off the project if it doesn't show up? Or is it alright if we leave it as it is? Thanks :)

**Comment by rudeboybert** [comment]
At think at this point in the semester given where we all are mentally, let's just leave things as is, and you won't be penalized for it.
